source/UpcommingMovies.UI/UpcommingMovies.UI/Models/MovieListItem.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Linq;
+using UpcommingMovies.Core.Domain.Models;
+
+namespace UpcommingMovies.UI.Models
+{
+    /// <summary>
+    /// Model for MainPage/MainPageViewModel
+    /// </summary>
+    public class MovieListItem
+    {
+        public MovieListItem(Movie movie)
+        {
+            Movie = movie;
+            Genres = string.Join(", ", movie.Genres.Select(g => g.Name));
+            HasGenres = movie.Genres.Any();
+            Image = movie.BuildBackdropImageUri(150) ?? movie.BuildPosterImageUri(150);
```

Comment:
The BuildBackdropImageUri and BuildPosterImageUri methods takes width as parameter. For the given width, it looks for nearest width available for image type (as supplied by configuration). I choose backdrop images as prefered image instead of poster by an arbitrary criteria, but now that you address this, I see that the minimum width for backdrop images is 300px, that is considerable larger than the 92px of the minimum width for posters. Sorry, my fault.